scripts/build-delphix-repository.sh

```diff
@@ -175,5 +175,10 @@ aptly repo add delphix-debs "$PKG_DIRECTORY"
 aptly publish repo -passphrase=delphix delphix-debs
 mkdir -p "$TOP/delphix-repo"
 mv "$HOME/.aptly" "$TOP/delphix-repo/"
+cat "$TOP/delphix-repo/aptly.config" <<-EOF
```

Comment:
(not your change, but we should fix this)
`cat "..."` --> `cat >"..."`

Reply:
should be fixed now.